src/models/image-map.ts

```diff
@@ -493,10 +493,12 @@ export const localAssetsImagesHandler: IImageHandler = {
     return !/:/.test(url) &&
            // or legacy firebase storage references
            !/^\/.+\/portals\/.+$/.test(url) &&
+           // spaces are not allowed
+           !/ /.test(url) &&
            // make sure there's at least one slash
            /\//.test(url) &&
            // make sure value ends with a file extension
-           /\.[a-z0-9]+$/i.test(url);
+           /\.[a-z]{3,}$/i.test(url);
```

Comment:
`.jpeg` is a valid image file extension, although not necessarily one we need to support.

Reply:
I can't think of any current image file extensions that contain digits, but there could be in the future. If this is ever used for audios or videos, it would rule out `.mp3` and `.mp4`.